package.json

```diff
@@ -29,9 +29,9 @@
     "files": [
         "dist"
     ],
-    "packageManager": "pnpm@7.27.0",
+    "packageManager": "pnpm@8.3.1",
     "scripts": {
-        "prChecks": "pnpm install && pnpm build && pnpm typecheck && pnpm lint && pnpm testRepo && pnpm docgen && pnpm buildSite",
+        "prChecks": "pnpm install && pnpm build && pnpm typecheck && pnpm lint && pnpm format && pnpm testRepo && pnpm docgen && pnpm buildSite",
```

Comment:
I'm not sure, since as I mentioned above, it could be disabled. Alternatively, we could run the same format check script as CI. Then outdated formatting would fail locally just like CI, instead of writing a file you might forget to commit.

Reply:
Yeah exactly, let's do that as mentioned above!